package.json

```diff
@@ -11,8 +11,6 @@
   "scripts": {
     "prebuild": "rimraf dist",
     "build": "nest build",
-    "ci:build": "npm run build && docker-compose build bull-monitor",
-    "ci:publish": "npm run ci:build && docker-compose push bull-monitor",
```

Comment:
I didn't see `ci:publish` getting used anywhere, and `ci:build` functionality is encapsulated in the build-push-action step